weaviate/connect/v4.py

```diff
@@ -568,7 +568,7 @@ async def get(
         return await self.__send(
             "GET",
             url=self.url + self._api_version_path + path,
-            params=params if params is not None else {},
+            params=params,
```

Comment:
Are you sure about this change?

Reply:
yes, basically we cannot have the param parameter set (`{}`counts as set) when there are optional parameters in the path. I cannot remember why we added an empty {} anymore :(